src/elements/content-sidebar/SidebarNavSignButton.tsx

```diff
@@ -1,18 +1,23 @@
 import * as React from 'react';
 import classnames from 'classnames';
 import { injectIntl, IntlShape } from 'react-intl';
+import SignIcon from '@box/blueprint-web-assets/icons/Medium/Sign';
+import { IconIconBlue, Size5 } from '@box/blueprint-web-assets/tokens/tokens';
+import { Tooltip as BPTooltip } from '@box/blueprint-web';
 import BoxSign28 from '../../icon/logo/BoxSign28';
 import Sign16 from '../../icon/fill/Sign16';
 import PlainButton, { PlainButtonProps } from '../../components/plain-button';
 // @ts-ignore Module is written in Flow
 import TargetedClickThroughGuideTooltip from '../../features/targeting/TargetedClickThroughGuideTooltip';
 import Tooltip, { TooltipPosition } from '../../components/tooltip';
+import { useFeatureConfig } from '../common/feature-checking';
```

Comment:
nit: can we not pass down the isPreviewModernizationEnabled as a prop from SideBarNav instead? The other components look to take this in as a prop.